archetype/simple-service/src/main/resources/archetype-resources/service/src/main/java/service/__name__Module.java

```diff
@@ -3,14 +3,24 @@
 #set( $symbol_escape = '\' )
 package ${package}.service;
 
-import ${package}.service.resource.HelloWorldResourceImpl;
-import ${package}.api.HelloWorldResource;
+import ${package}.service.resource.HelloWorldResource;
 import com.google.inject.PrivateModule;
 
 public class ${name}Module extends PrivateModule {
 
   protected void configure() {
-    bind(HelloWorldResource.class).to(HelloWorldResourceImpl.class);
+    bind(HelloWorldResource.class);
+
+    // Note: If any clients will be used that directly provide injection of JAX-RS interfaces, then
+    // they should be installed in this private module to avoid registering client JAX-RS resources
+    // with the server.
+    //
+    // It's also worth noting that using any multibinders across both private and normal Guice
+    // modules is not a supported feature of Guice. Therefore this module should be your only
+    // private module, if needed at all. Prefer standard Guice modules for installing Beadledom
+    // clients if possible.
+    //
+    // The Correlation ID header name for clients could also be configured/overridden here.
```

Comment:
Is this actually true? The correlation ID uses an optional binding, so I don't think it will play well with this being a `PrivateModule`.

Reply:
I think it might actually be true if this is the only `PrivateModule` used.

Reply:
I don't know. This wasn't part of the code that I touched, but ill look into it

Reply:
Changed guidance to make users aware that this will only work with one private module, and added updated our recommended Client module creation documentation: https://github.com/cerner/beadledom/pull/133/commits/b3519fab9fe1880bdda0e7948a61801642e70cd8